Dockerfile.backend

```diff
@@ -3,16 +3,10 @@ FROM openbuildservice/base
 # FIXME: https://bugzilla.opensuse.org/show_bug.cgi?id=957818
 RUN rm -rf /var/cache/zypp/*
 
-# Install requirements for running the backend & signd
-RUN zypper -n install --no-recommends --replacefiles \
-  obs-server obs-signd \
-  obs-service-download_src_package obs-service-download_files \
-  obs-service-download_url \
-  obs-service-format_spec_file obs-service-kiwi_import \
-  perl-Devel-Cover \
-  osc
+RUN /root/bin/docker-bootstrap.sh backend
 
 # Add our sign configuration
+RUN zypper -n install --no-recommends --replacefiles obs-signd
```

Comment:
💭 Why is this not part of the bootstrap script?

Reply:
Moved to the bootstrap script.